common/src/main/java/com/microsoft/identity/common/internal/controllers/BrokerOperationExecutor.java

```diff
@@ -181,7 +181,7 @@ private <U> void emitOperationSuccessEvent(@NonNull final BrokerOperation<U> ope
         final String telemetryApiId = operation.getTelemetryApiId();
         if (telemetryApiId != null) {
             final ApiEndEvent apiEndEvent = new ApiEndEvent();
-            final ApiEndEvent successEvent = apiEndEvent
+            final ApiEndEvent successEvent = (ApiEndEvent) apiEndEvent
```

Comment:
Why is this casting needed?

Reply:
The chaining `.putApiId().isApiCallSuccessful()` returns the base class, so we need to cast.